physical/rafttypes.go

```diff
@@ -0,0 +1,18 @@
+package physical
+
+import (
+	"context"
+	"crypto/tls"
+	"net"
+
+	"github.com/hashicorp/vault/vault/cluster"
```

Comment:
Not necessary now, but the fact that `vault` reaches into `physical` and now `physical` reaches into `vault/cluster` makes me think we should maybe move `vault/cluster`.